### ISSUE TITLE
Update README to note the version of nvidia-docker

### ISSUE THREAD

**Issue by YunzhuLi** [RobotLocomotion/spartan#201]
Referring from this issue [https://github.com/NVIDIA/nvidia-docker/issues/534](https://github.com/NVIDIA/nvidia-docker/issues/534) that we need to install nvidia-docker 1. The newer version, nvidia-docker 2, will result in the following errors
```
libGL error: No matching fbConfigs or visuals found
libGL error: failed to load driver: swrast
```

It might be better if we can note this in the README.

Thanks.

**Comment by manuelli** [comment]
Done

**Comment by EricCousineau-TRI** [comment]
FTR I am able to run `glxgears` using the `spartan.Dockerfile` (or just `libglvnd`) on Ubuntu 18.04 using `nvidia-docker-2`. Seems like it's perhaps fixed now, at least on 18.04?
